src/mesh/tdymesh.c

```diff
@@ -125,20 +163,29 @@ PetscErrorCode TDyMeshGetVertexInternalCells(TDyMesh *mesh,
   return 0;
 }
 
-/// Given a mesh and a vertex index, retrieve an array of associated subcell
-/// indices and their number.
+/// Given a mesh and a vertex index, return number of associated internal
```

Comment:
Is an "internal" cell a cell not attached to a domain boundary? Or does this include all cells not attached to parallel subdomain boundaries?

Reply:
"Internal" cell is my lingo and here is the explanation. Boundary faces have an internal cell (DMPlex id>=0) and a fictitious cell (DMPlex id<0).

We probably need to add comments in the code that explicitly states our lingo.